assets/themes/zeppelin/css/style.css

```diff
@@ -88,32 +99,41 @@ body {
 .navbar {
   background: #3071a9;
   border-bottom: 0px;
-  height: 50px;
+  height: 60px;
+  -webkit-box-shadow: 0 8px 6px -6px #999;
```

Comment:
One difference between #2371 and this PR is that 

- the background of the main content is white #2371 
- the background color for the main content of the official homepage is blue.